server/datastore/mysql/software.go

```diff
@@ -119,23 +119,54 @@ func nothingChanged(current, incoming []fleet.Software, minLastOpenedAtDiff time
 	return true
 }
 
+func listSoftwareByHostIDShort(
```

Comment:
@michalnicp How do you feel about pulling this up to the data-store interface? Right now I'm also in the need of querying the database for 'all the software for a given host id', we have some methods for that, but all of them perform a bunch of unnecessary joins.

Reply:
That seems reasonable to me.